grammars/erlang.bnf

```diff
@@ -321,6 +324,34 @@ argument_definition_list ::= '(' argument_definition? (',' argument_definition)*
 clause_guard ::= when <<guarded guard>> {pin=1}
 clause_body ::= '->' exprs {pin=1}
 
+app_expression ::= '{' 'application' ',' module_ref ',' app_parameters '}' {elementType=tuple_expression}
```

Comment:
Because in most cases applications have module which starts all processes and do primary job, and name of this module is the same as application name. 
Even if some projects are different, we just won't resolve it.